.eslintrc.js

```diff
@@ -329,5 +330,20 @@ module.exports = {
         'google-camelcase/google-camelcase': 0,
       },
     },
+    {
+      'files': ['examples/**/*.js', '!examples/visual-tests/**/*.js'],
```

Comment:
Since you are linting the visual diff files, you'd have to remove `'!examples/visual-tests/**/*.js'` from here, right?

Reply:
It's two different sets of rules. My reasoning is that `examples/` code is very loose (e.g. example third-party code like on `amp-script/`), whereas `visual-tests` should have some additional rules enforced, e.g.:

https://github.com/ampproject/amphtml/blob/e8ff7f185a969bad499781dab35a43df1802be8c/.eslintrc.js#L336-L338

I'd be happy to restructure into more granular rulesets if needed.

Reply:
Added a suggestion above. Would that work?